polus-recycle-vector-plugin/README.md

```diff
@@ -13,7 +13,9 @@ This plugin uses [filename patterns](https://github.com/USNISTGOV/MIST/wiki/User
 
 All filename patterns must include `x` and `y` grid positions for each image or a sequential position `p`, but not both. This differs from MIST in that `r` and `c` are used to indicate grid row and column rather than `y` and `x` respectively. This change was made to remain consistent with Bioformats dimension names and to permit use of `c` as a channel variable.
 
-In addition to the position variables (both `x` and `y`, or `p`), the only other variables that can be used are `r`, `z`, `c`, and `t`. Images with the same `r`, `z`, and `t` will have the same stitching vector applied to them.
+In addition to the position variables (both `x` and `y`, or `p`), the only other variables that can be used are `r`, `z`, `c`, and `t`.
+
+**Version 1.1 Update:** If no inputs are provided in the `groupBy` variable, then all non-positional variables that vary from the stitching vector will cause a new stitching vector to be generated. For example, if replicate, x, and y are all contained within a stitching vector but the vector contains only files for channel 1 (e.g. r{rrr}_x{xxx}_y{yyy}_c001.ome.tif), then the stitching vector filepattern can be set to r{rrr}_x{xxx}_y{yyy}_c{ccc}.ome.tif and `--groupBy r` can be used to recycle the stitching vector for additional channels.
```

Comment:
Dang, I just finished the review of the bfio. Here you introduce another variable, r... which I believe would be analogous to series in the bioreader class you have in bfio. However, you drop this entirely in biowriter and use a different variable here. I am fine with either, but I do think we should be consistent between these two libraries.

Reply:
`r` is not analagous to series. Since WIPP has explicitly made the decision to only handle positional data within individual files, the `r` variable is really for grouping data only. Same with the `t` and `c` values.